BinaryTree.java

```diff
@@ -0,0 +1,198 @@
+package com.katermar.assignment2;
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * Created by katermar on 5/23/2017.
+ */
+public class BinaryTree {
+    TreeNode root;
+
+    /*Inner class for Binary Tree node
+    * @field value stores integer value
+    * @field left, right store left and right subtrees*/
+    public static class TreeNode {
+        private final int value;
+        private TreeNode left, right;

```

Comment:
Declaring multiple fields or variables in a single line is more difficult to read than a single declaration per line. Could you please change code to split declarations of left and right?

The same applies to code from assignment 1, in which I missed it. You could change it there as well or leave as is.